vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java

```diff
@@ -772,26 +594,19 @@ public VLoadingIndicator getLoadingIndicator() {
      */
     @Deprecated
     public boolean isLoadingIndicatorVisible() {
-        return getLoadingIndicator().isVisible();
+        return false;
     }
 
     private void addVariableToQueue(String connectorId, String variableName,
             Object value, boolean immediate) {
-        boolean lastOnly = !immediate;
-        // note that type is now deduced from value
-        serverRpcQueue.add(new LegacyChangeVariablesInvocation(connectorId,
-                variableName, value), lastOnly);
-        if (immediate) {
-            serverRpcQueue.flush();
-        }
     }
```

Comment:
It seems that this method can be removed together with all `updateVariable` methods.